designs/sam-config.md

```diff
@@ -265,7 +252,7 @@ Implementation
 CLI Changes
 -----------
 
-New command line argument is added per command called `--env` to be able to specify non default environment section within a config file.
+New command line argument is added per command called `--config-prefix` to be able to specify non default environment section within a config file.
```

Comment:
Can you make a note somewhere that says --config-file prefix is not implemented yet?

Reply:
Done